Documentation/ApiOverview/Fal/Architecture/Database.rst

```diff
@@ -83,6 +83,39 @@ The :sql:`sys_file_metadata` table is extended by the system extension
 `filemetadata`_. In particular, it adds the necessary definitions
 to categorize files with :ref:`system categories <categories>`.
 
+Also some other helpful metadata attributes are filled, most of them
```

Comment:
`filled`? What do you think about `provided`?

Reply:
Ahh. I think I got it. Yes, `width` and `height` and maybe `creater_tool` will be filled automatically, but most of the other fields are not filled.

Reply:
You are right, "filled" is not right here. I changed it! :)